PackingList/app/src/main/java/com/example/packinglist/MainMenuActivity.kt

```diff
@@ -52,9 +95,16 @@ class MainMenuActivity : AppCompatActivity() {
      * @return no direct return. The callback in Response.Listener will fire when the response comes back.
      */
     fun apiButtonPressed(@Suppress("UNUSED_PARAMETER") view: View) {
-        // the url we are calling for the GET request
-        val latitude = 33.209839
-        val longitude = -87.569168
+        // the url we are calling for the GET request. The Latitude and Longitude comes from the Geo data in SharedPreferences
+        val sharedPreferences = App.context.applicationContext.getSharedPreferences(Constant.appSharedPreferencesKey, Context.MODE_PRIVATE)
+        var latitude = sharedPreferences.getString("LOCATION_LAT", null)
+        var longitude = sharedPreferences.getString("LOCATION_LON", null)
+
+        //if the geo data doesn't exist in shared preferences, then use Tuscaloosa as default
+        if(latitude == null || longitude == null) {
+            latitude = "33.204095"
+            longitude = "-87.544453"
```

Comment:
Let's extract these into constants so it's easier to read and we don't need the comment above
```suggestion
            latitude = Constant.tuscaloosaLatitude
            longitude = Constant.tuscaloosaLongitude
```

Reply:
good Idea!